components/home/SpacesOverview/spaces-overview.tsx

```diff
@@ -27,14 +26,14 @@ interface SpaceDetailed extends Space {
  * @returns list of spaces
  */
 async function getUserSpaces() {
-    const url = `${process.env.NEXT_PUBLIC_API}/v1/spaces/?page=1&page_size=4&ordering=-updated_at`; // For the future load automatically on scrolling
+    const url = `${process.env.NEXT_PUBLIC_API}/v1/spaces/?page=1&ordering=-updated_at`; // For the future load automatically on scrolling
```

Comment:
here I want to keep the page results only to 4 spaces for now
```suggestion
    const url = `${process.env.NEXT_PUBLIC_API}/v1/spaces/?page=1&page_size=4&ordering=-updated_at`; // For the future load automatically on scrolling
```